jr-extension-javatime/src/main/java/com/fasterxml/jackson/jr/extension/javatime/date/DateValueReader.java

```diff
@@ -0,0 +1,25 @@
+package com.fasterxml.jackson.jr.extension.javatime.date;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.jr.ob.api.ValueReader;
+import com.fasterxml.jackson.jr.ob.impl.JSONReader;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateValueReader extends ValueReader {
+    protected DateValueReader() {
+        super(Date.class);
```

Comment:
Technically this is not JAva 8 date/time type... so probably should be in core `jr-objects`?

But also, it needs to support deserialization from Integer numbers (timestamp).